src/animation/AnimationUtils.js

```diff
@@ -158,9 +158,59 @@ var AnimationUtils = {
 
 		}
 
+	},
+
+	clone: function ( source ) {
+
+		var sourceLookup = new Map();
+		var cloneLookup = new Map();
+
+		var clone = source.clone();
+
+		parallelTraverse( source, clone, function ( sourceNode, clonedNode ) {
+
+			sourceLookup.set( clonedNode, sourceNode );
+			cloneLookup.set( sourceNode, clonedNode );
+
+		} );
+
+		clone.traverse( function ( node ) {
+
+			if ( ! node.isSkinnedMesh ) return;
+
+			var clonedMesh = node;
+			var sourceMesh = sourceLookup.get( node );
+			var sourceBones = sourceMesh.skeleton.bones;
+
+			clonedMesh.skeleton = sourceMesh.skeleton.clone();
+			clonedMesh.bindMatrix.copy( sourceMesh.bindMatrix );
+
+			clonedMesh.skeleton.bones = sourceBones.map( function ( bone ) {
+
+				return cloneLookup.get( bone );
+
+			} );
+
+			clonedMesh.bind( clonedMesh.skeleton, clonedMesh.bindMatrix );
```

Comment:
Copying matrix to `.bindMatrix` is done in `SkinnedMesh.bind()`.

https://github.com/mrdoob/three.js/blob/dev/src/objects/SkinnedMesh.js#L112

So I think you can replace this line with

    clonedMesh.bind( clonedMesh.skeleton, sourceMesh.bindMatrix );

and remove the line above `clonedMesh.bindMatrix.copy( sourceMesh.bindMatrix );`.

Reply:
Done ✅

Reply:
You need to pass `"source"Mesh.bindMatrix`.

Reply:
Fixed ✅